Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift

```diff
@@ -190,10 +189,10 @@ public struct MessageField: View {
         }
         .sheet(isPresented: $isGIFPickerPresented) {
             if let giphyKey = configuration.giphyKey {
-                GiphyPicker(giphyKey: giphyKey)
-                    .ignoresSafeArea()
-                    .presentationDetents([.fraction(0.9)])
-                    .presentationDragIndicator(.hidden)
+                GiphyPicker(giphyKey: giphyKey, giphyConfig: configuration.giphyConfig)
+                        .ignoresSafeArea()
+                        .presentationDetents([.fraction(configuration.giphyConfig.presentationDetents)])
+                        .presentationDragIndicator(.hidden)
```

Comment:
```suggestion
                GiphyPicker(
                    giphyKey: giphyKey, 
                    giphyConfig: configuration.giphyConfig
                )
                .ignoresSafeArea()
                .presentationDetents(
                    [.fraction(configuration.giphyConfig.presentationDetents)]
                )
                .presentationDragIndicator(.hidden)
```

Reply:
I left the minor suggestion about code convention.

Reply:
cool can we merge this

Reply:
Of course!